astropy/units/quantity_helper/converters.py

```diff
@@ -206,10 +206,8 @@ def converters_and_unit(function, method, *args):
                 # _can_have_arbitrary_unit failed: arg could not be compared
                 # with zero or checked to be finite. Then, ufunc will fail too.
                 raise TypeError(
-                    "Unsupported operand type(s) for ufunc {}: '{}'".format(
-                        function.__name__,
-                        ",".join([arg.__class__.__name__ for arg in args]),
-                    )
+                    f"Unsupported operand type(s) for ufunc {function.__name__}:"
+                    f" '{','.join(arg.__class__.__name__ for arg in args)}'"
```

Comment:
I think I'd write `+ ",".join(....)` here instead of an f-string that does little else.

Reply:
Yeah the string wrap didn't do so great there did it?

Reply:
Given that your suggestion would change the string, do you want to keep it is how it currently is or do you want me to do something else?

Reply:
No, it wouldn't change the string: it would be in full,
```
f"Unsupported operand type(s) for ufunc {function.__name__}: "
+ ",".join(arg.__class__.__name__ for arg in args)
```

But I'm also fine with just keeping the original here.

Reply:
That would change the API because the quotes around the string produced by the join will be missing.

Reply:
Ah, you're right. Maybe just revert then.

Reply:
@mhvk I reverted the above change. Does that work for you?